### ISSUE TITLE
Build Node Page: Add instructions on how to set up a mesh node

### ISSUE THREAD

**Issue by dcwalk** [tomeshnet/tomesh.net#101]
Difficult right now to find how to get a mesh node up...

Thoughts:
- Make clear from 'get involved' and elsewhere on site what equipment we have
- Have instructions on the website? (currently: in [tomesh.101/tutorial](https://github.com/tomeshnet/tomesh.101/tree/master/tutorial), [groundupworks presentation](http://home.groundupworks.com:9090/#13)
- Separate page with instructions and hardware, used as part of virtual mesh night?

**Comment by garrying** [comment]
Looking at this as well: https://github.com/tomeshnet/prototype-cjdns-pi2/blob/master/README.md

**Comment by benhylau** [comment]
I would prefer keeping docs to the code repository itself, and have website link to it rather than duplicating information? Unclear parts can be clarified in the README.md itself.

**Comment by ana0** [comment]
I'll wait on this for the time being for more discussion!

I understand the goal of not duplicating information and keeping the tutorial in an easy to update place for sure . . .  I suggested adding some "build a node" information to the main site because of my recent experience as a newcomer to tomesh - my first question when I came to the site was, "How do I build a node, what tools are they using?"  The meta-questions were "Why this set of tools, and what's the state of the mesh?"  

The answers are a bit buried, from a ui perspective - you have to click through to another site (github) and look through the repos.  I don't know if my first questions are typical, but I know my desire to get involved and find out the answers was considerably more than we can count on from other newcomers.  I think at least direct links to the relevant files/readmes, placed somewhere prominent would be a good addition

**Comment by benhylau** [comment]
@ana0 Your concern regarding information being buried is definitely typical. Many people have expressed this. All the info is buried in Wekan and various Github repo atm. I wonder if a **Projects** tab to feature the long-running projects is appropriate?

For example, sudoroom.org has a tab linking to their [list of projects](http://sudoroom.github.io/sudoRoom-Projects-Landing-page/). Maybe we can have a tab like that on our website?

Ours are basically our **Long Running** cards on https://wekan.tomesh.net/b/LWS8X7sGFXqDgZ7ag/tomesh-net

- Reading Networks
- Raspberry Pi & cjdns based mesh image + weekly virtual mesh
- Website
- Mesh ISP
- Outernet

I guess these can belong there too:

- Steamlink
- DIY WiFi antennas

UX-wise, clicking into a tab, and then a particular project, to reach the long-form description of setting up a node, the reasons behind choice of software and hardware, how the biweekly virtualmesh event is related, and the corresponding project channel at chat.tomesh.net... idk if that's one click too many? But it does allow for multiple projects to be featured equally. Thoughts?

**Comment by dcwalk** [comment]
Hey, wanted to chime in on this:

1. I think having instructions all in one place is a priority, and we don't have that in practice. It could be github, but we aren't there in practice (VPN and Slidedeck for Peering come to mind). Do you see those going in the github readme @benhylau ? Because that is where I could see the value of a webpage, linking to the github for the repo then the Peering and VPN instructions on a webpage (and link to that riseup pad to add keys)
2. I'm not sure having a "Projects" tab is the best way to handle the information being buried -- I think we would have to prototype it to understand that flow better
3. Do all those projects have to be featured equally? Looking at the list the `Events` section covers 2 of 'em (virtual mesh and reading networks), the website improvement feels more conceptual anyhow...

I like the first list of content for the website you mentioned @benhylau 
> the long-form description of setting up a node, the reasons behind choice of software and hardware, how the biweekly virtualmesh event is related, and the corresponding project channel at chat.tomesh.net

But almost feel like "Get on the Mesh" or "Build a Node" is starting to become a focus and maybe we want to emphasize it?

I agree with @ana0 what we want it somewhere prominent?  (Maybe "contact" page can be de-emphasized as we have it in global footer and via get involved)

**Comment by benhylau** [comment]
>But almost feel like "Get on the Mesh" or "Build a Node" is starting to become a focus and maybe we want to emphasize it?

I was actually hoping to de-emphasize that :P as some may be interested in other parts? And it'd be nice to have more people involved in say... Website, Outernet, etc.

**Comment by dcwalk** [comment]
Fair point 😄

**Comment by darkdrgn2k** [comment]
the BUILD A NODE page has now been mearged #169 that will pull the prototype readme file.
